File: BEE 1079.py
n = int(input())

for i in range(n):
    num1, num2, num3 = map(float, input().split())
    avg = ((num1 * 2) + (num2 * 3) + (num3 * 5)) / (2 + 3 + 5)
    print("{:.1f}".format(avg))

File: BEE 1079.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
// var lines = input.split('\n');
var sep = [' ', '\n'];
// console.log(separators.join('|'));
var line = input.split(new RegExp(sep.join('|'), 'g'));

var n = parseInt(line.shift());

for (let i = 0; i < n; i++){
    var num1 = parseFloat(line.shift());
    var num2 = parseFloat(line.shift());
    var num3 = parseFloat(line.shift());
    var avg = ((num1 * 2) + (num2 * 3) + (num3 * 5)) / (2 + 3 + 5);
    console.log(avg.toFixed(1));
}